client/components/layout/Sunflower.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+
+import Logo from '../../styles/images/sunflower_white.svg';
+
+const Sunflower = () => {
+  return (
+    <Box
+      component="img"
+      sx={{
+        height: 48,
```

Comment:
Consider passing width/height in as props so we can resuse the component with different sizing